launch/hardware/dynamixels.launch

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<launch>
+
+    <arg name="machine" default="localhost"/>
+
+    <node name="dynamixel_manager" pkg="dynamixel_controllers" type="controller_manager.py" required="true">
+        <rosparam>
+            namespace: dxl_manager
+            serial_ports:
+                pan_tilt_laser_port:
+                    port_name: "/dev/ttyUSB0"
```

Comment:
This launch file could be more generic in my opinion. Robot specific parameters should be store in launch file in bringup folder and passed through.